marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -402,8 +403,9 @@ public void clearUserModules(List<String> resourceNamesToNotDelete) {
             runInDatabase(query, hubConfig.getDbName(DatabaseKind.MODULES));
         } catch (Exception e) {
             logger.error("Failed to clear user modules, cause: " + e.getMessage(), e);
+            throw new RuntimeException(e);
```

Comment:
Let's drop the logger command above instead include the message in the first arg to RuntimeException.